website/config.toml

```diff
@@ -8,4 +8,4 @@ publishDir = "site"
 [params]
   description = "**[Ace](/ace)**, an accessibility checker for EPUB!  \n\
   A **[DAISY Consortium](http://daisy.org)** project. \n\
-  Latest version: v1.0.2"
+  Latest version: v1.0.3"
```

Comment:
At some point we'll need to do this at build time… but let's not tackle this in this PR ;-)

Reply:
okay

Reply:
Done: https://github.com/daisy/ace/pull/226/commits/1413f464790c75baf79e50f56cc754d0e55be819